client/flutter/lib/pages/settings_page.dart

```diff
@@ -127,6 +133,45 @@ class _SettingsPageState extends State<SettingsPage>
                         .homePagePageSliverListSettingsNotificationsInfo,
                     isToggled: _notificationsEnabled ?? false,
                     onToggle: _toggleNotifications),
+                SizedBox(
+                  height: 20,
+                ),
+                divider,
+                menuItem(
+                  title: S.of(context).homePagePageSliverListShareTheApp,
+                  onTap: () {
+                    FirebaseAnalytics().logShare(
+                        contentType: 'App',
+                        itemId: null,
+                        method: 'Website link');
+                    Share.share(
+                      S.of(context).commonWhoAppShareIconButtonDescription,
+                      sharePositionOrigin:
+                          Rect.fromLTWH(0, 0, size.width, size.height / 2),
+                    );
+                  },
+                ),
+                divider,
+                // TODO: Localize
+                menuItem(
+                    title: 'Provide app feedback',
+                    onTap: () {
+                      FirebaseAnalytics().logEvent(name: 'Feedback');
+                      // TODO: Implement feedback #989 #1015
+                    }),
+                divider,
+                menuItem(
+                  title: S.of(context).homePagePageSliverListAboutTheApp,
+                  onTap: () {
+                    FirebaseAnalytics().logEvent(name: 'About');
+                    return Navigator.of(context)
+                        .push(MaterialPageRoute(builder: (c) => AboutPage()));
```

Comment:
Are we still using `MaterialPageRoute` in the Cupertino app? Do we want to consider switching to whatever new routing / linking scheme we end up using for the app here, too?

Reply:
Eventually yes, but I think for now this gives us a platform-comfortable feel.  We can change all of them once David makes a decision.